content/en/data_streams/dotnet.md

```diff
@@ -27,11 +27,18 @@ environment:
   - DD_DATA_STREAMS_ENABLED: "true"
 ```
 
+#### Additional Azure Service Bus Setup
```

Comment:
can we move this section below the 'Monitoring SNS-to-SQS pipelines' section, and use the same header font as the others? (we get more questions from customers on SQS and SNS than Azure Service Bus)

Reply:
I'll need to verify that the link in the table still works, but applied in 2a8d53cd1765568ad76b1a228b325f7de7dd7352

Reply:
Yep the link to the revised section works, resolving